directives/point_of_interest.py

```diff
@@ -90,11 +89,23 @@ def run(self):
             # included in the A+ course materials (e-book).
             return []
 
-        # add an extra div to force content to desired height
-        hcontainer_opts = {
-            u'style':'height:'+self.options.get('height', '')+';',
-            u'class': container_class +' poi-content row',
-        }
+        if 'no_poi_box' not in self.options:
+            container_class = 'poi-container'
+            content_name = name + '-content'
+
+            # add an extra div to force content to desired height
+            hcontainer_opts = {
+                u'style': 'height:' + self.options.get('height', '') + ';',
+                u'class': container_class + ' poi-content row',
+            }
+        else:
+            container_class = 'no_poi_box'
+            content_name = name
+            hcontainer_opts = {
```

Comment:
No need to repeat identical `hcontainer_opts` definition twice. Define it outside the if statement since it is not affected by the if condition (the option `no_poi_box`).

Reply:
It is affected by the if condition, compare the lines 99 and 106. In the else condition `u'class` has only `row` as a value.